lib/hydra/works/models/concerns/collection_behavior.rb

```diff
@@ -22,6 +22,39 @@ module CollectionBehavior
       type [RDFVocabularies::PCDMTerms.Collection,WorksVocabularies::WorksTerms.Collection]
       include Hydra::Works::AggregatesGenericWorks
       include Hydra::Works::AggregatesCollections
+      include Hydra::Works::BlockChildObjects
+
+      filters_association :members, as: :child_collections, condition: :works_collection?
+      filters_association :members, as: :child_generic_works, condition: :works_generic_work?
+    end
+
+    module ClassMethods
+      def indexer
+        Hydra::Works::CollectionIndexer
+      end
+    end
+
+    # @return [Boolean] whether this instance is a Hydra::Works Collection.
+    def works_collection?
+      true
+    end
+
+    # @return [Boolean] whether this instance is a Hydra::Works Generic Work.
+    def works_generic_work?
```

Comment:
I was following the pattern set at pcdm using pcdm_collection? and pcdm_object.  I do not have a problem using just generic_work?  It seems intuitive enough.  I would like to see consistency.  What happens if sufia defines a generic_work?  In the current pattern, there would be works_generic_work?  and sufia_generic_work?